### PR TITLE
chore: add translations for 1.9.0

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This updates the translations needed for our 1.9.0 release.  There are two ways to update translations.
 - You can add suggestions for each translated string. [click here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) for more information on adding suggestions.
 - You can update translations using one of the csvs below by editing the file and re-attaching it to this pull request
    - **de**: [de-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961725/de-locale-update-1.9.0.csv)
    - **es**: [es-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961726/es-locale-update-1.9.0.csv)
    - **fr**: [fr-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961727/fr-locale-update-1.9.0.csv)
    - **it**: [it-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961728/it-locale-update-1.9.0.csv)
    - **ja**: [ja-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961729/ja-locale-update-1.9.0.csv)
    - **ko**: [ko-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961730/ko-locale-update-1.9.0.csv)
    - **pt-br**: [pt-br-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961731/pt-br-locale-update-1.9.0.csv)
    - **zh-cn**: [zh-cn-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14961732/zh-cn-locale-update-1.9.0.csv)

**Testing**
 - [x] Verify all strings are captured by running the translation check (https://github.com/sassoftware/vscode-sas-extension/actions/runs/8879422866)
 - [x] Verify **de** strings are translated
 - [x] Verify **es** strings are translated
 - [x] Verify **fr** strings are translated
 - [x] Verify **ja** strings are translated
 - [x] Verify **ko** strings are translated
 - [x] Verify **zh-cn** strings are translated
 - [x] Verify **it** strings are translated
 - [x] Verify **pt-br** strings are translated

### CODE REVIEW

File: l10n/bundle.l10n.zh-cn.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "An error was encountered when loading table data. This usually happens when a table is too large or the data couldn't be processed. See console for more details.": "An error was encountered when loading table data. This usually happens when a table is too large or the data couldn't be processed. See console for more details.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:2]:
```suggestion
  "An error was encountered when loading table data. This usually happens when a table is too large or the data couldn't be processed. See console for more details.": "加载表数据时遇到错误。 当表太大或无法处理数据时，通常会发生这种情况。 请参阅控制台了解更多详细信息。",
```

**Comment by SASJedi** [comment]
[pt-br-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14978691/pt-br-locale-update-1.9.0.csv)

**Comment by meilanji** [comment]
Added Korean translation.
[ko-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14986251/ko-locale-update-1.9.0.csv)

**Comment by Criptic** [comment]
[de-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/15068696/de-locale-update-1.9.0.csv)

**Comment by dfxvir** [comment]
[fr-locale-update-1.9.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/15149082/fr-locale-update-1.9.0.csv)

**Pull request merged**